packages/forms-web-app/src/controllers/document.js

```diff
@@ -73,6 +80,59 @@ const getDocument = async (req, res) => {
 	}
 };
 
+/**
+ * links user to a document, requires an active session
+ * currently only used in links to pdf for has and s78 submission
+ * @type {import('express').Handler}
+ */
+const getDocumentV2 = async (req, res) => {
+	const { appealOrQuestionnaireId, documentId } = req.params;
+
+	if (appealOrQuestionnaireId.includes('../') || documentId.includes('../')) {
```

Comment:
Why would these things have that in them?

Reply:
They _shouldn't_ have them in, but in theory an appellant could set these params and try and instigate a path traversal attack.

Reply:
I'll be honest I'm on the side of checking they are uuids...
also maybe best to do in the handler

Reply:
Yeah, seems safer than trying to do string includes or regex, so have added a check in the fetch step